base_model_restrict_update/models/ir_model_access.py

```diff
@@ -48,8 +49,13 @@ def _test_restrict_update(self, model):
 
     @api.model
     def _readonly_exclude_models(self):
-        """ Models updtate/create by system, and should be excluded from checking """
-        return self.sudo().search([
+        transient_models = self.env["ir.model"].sudo().search(
+            [("transient", "=", True)]).mapped("model")
+        skipped_models = self.env["ir.model"].sudo().search([
+            ("skip_check_for_readonly_users", "=", True)
+        ]).mapped("model")
+        # Models updtate/create by system, and should be excluded from checking
+        return skipped_models + transient_models + self.sudo().search([
```

Comment:
```suggestion
        skipped_models = self.env["ir.model"].sudo().search(
            ["|", ("transient", "=", True), ("skip_check_for_readonly_users", "=", True)]
        ).mapped("model")
        # Models updtate/create by system, and should be excluded from checking
        return skipped_models + self.sudo().search([
```
Let's combine the searches into one. ;)

Reply:
committed, thanks